client/core/core.go

```diff
@@ -1168,6 +1170,11 @@ func New(cfg *Config) (*Core, error) {
 		return nil, err
 	}
 
+	var seedGenerationTime uint64
+	if seedGenTime, err := boltDB.SeedGenerationTime(); err == nil {
+		seedGenerationTime = seedGenTime
+	}
+
```

Comment:
I would prefer if you didn't ignore the error. You could return a typed error for bucket not existing, and document that an error is expected for legacy wallets, or just return zero and no error, I guess. I would actually prefer a db upgrade, but maybe that's overkill? Are there any other bucket keys in the db that may or may not exist?